api/graphql/schema.graphql

```diff
@@ -1571,6 +1593,24 @@ input WorkExperienceInput {
   endDate: Date @rename(attribute: "end_date")
   details: String
   skills: UpdateExperienceSkills @spread
+  employmentCategory: EmploymentCategory
+    @rename(attribute: "employment_category")
+  extSizeOfOrganization: ExternalSizeOfOrganization
+    @rename(attribute: "ext_size_of_organization")
+  extRoleSeniority: ExternalRoleSeniority
+    @rename(attribute: "ext_role_seniority")
+  govEmploymentType: GovEmploymentType @rename(attribute: "gov_employment_type")
+  govPositionType: GovPositionType @rename(attribute: "gov_position_type")
+  govContractStartDate: Date @rename(attribute: "gov_contract_start_date")
+  govContractEndDate: Date @rename(attribute: "gov_contract_end_date")
+  govContractorRoleSeniority: GovContractorRoleSeniority
+    @rename(attribute: "gov_contractor_role_seniority")
+  govContractorType: GovContractorType @rename(attribute: "gov_contractor_type")
+  cafEmploymentType: CafEmploymentType @rename(attribute: "caf_employment_type")
+  cafForce: CafForce @rename(attribute: "caf_force")
+  cafRank: CafRank @rename(attribute: "caf_rank")
+  classification: UUID @rename(attribute: "classification_id")
+  department: UUID @rename(attribute: "department_id")
```

Comment:
Maybe these two should be something like DepartmentBelongsTo?

Reply:
Hm, I think I should have "id" in the name of that, changed in https://github.com/GCTC-NTGC/gc-digital-talent/pull/11947/commits/371b58a401f249f4cac10d64b3ee9b20b29484a9
Not sure about sticking "BelongsTo" in there

Reply:
The reason I am hesitant about trying to keep it consistent is because it isn't the same at all. We're storing values in JSON and I had to install something to mimic normal Eloquent relations when querying the experience 😕

Reply:
Given that you did install something to mimic normal eloquent relations... would DepartmentBelongsTo actually work?

Reply:
It doesn't seem like it'll work without issue/refactor

Schema change

![image](https://github.com/user-attachments/assets/39ccf4f5-f6d3-4dd6-a675-22b0f8852bd7)

Test input is then

![image](https://github.com/user-attachments/assets/408ea64a-bf2c-4c6d-825b-62b341c96d50)

Resulting in array to string error

![image](https://github.com/user-attachments/assets/fa25ffd7-5186-4419-bbc2-6e9354b2b86b)

Adding this logging

![image](https://github.com/user-attachments/assets/1d079933-271d-4f3f-afee-dbfbc006dae1)

Reveals it is passing in the belongs to object and not handling it behind the scenes

![image](https://github.com/user-attachments/assets/1126ba2c-b481-461e-a36f-70a05d83ed44)

Reply:
Hang on, hang on.
I got and tried an idea

Using the existing, handy `@pluck` in https://github.com/GCTC-NTGC/gc-digital-talent/pull/11947/commits/53cd3238adc16b2963b32a5c67843971f9d3a2da

Reply:
Hmm, this doesn't really work the same way.  For example, you can't use disconnect anymore.  I might play with this a bit today to see if I can figure it out.

Reply:
Bah, I can't get this to work either.  Oh well, I think you've taken this as far as we can.  I'm closing this.